### PR TITLE
Add personal keymaps

### PR DESCRIPTION
**Pull request by ALysaght**
Add primarily 36/46 key keymaps

## Description

Preonic, Atreus62, dz65, nyquist, user files

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/ortho36/config.h [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#define DIODE_DIRECTION COL2ROW
+
+#define RGB_DI_PIN D3
+#define RGBLED_NUM 11
+#define RGBLIGHT_ANIMATIONS
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+#endif
```
Comment (ridingqwerty) [inline keyboards/handwired/ortho36/config.h:46]:
```suggestion
```

Use of `#pragma once` above obviates need for this `#endif`

File: keyboards/handwired/ortho36/config.h [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xFEED
+#define PRODUCT_ID 0x0000
+#define DEVICE_VER 0x0001
+#define MANUFACTURER    alysaght
+#define PRODUCT         ortho36
+#define DESCRIPTION     handwired mini-planck
```
Comment (ridingqwerty) [inline keyboards/handwired/ortho36/config.h:25]:
```suggestion
```

Omit description field.

**Review by ridingqwerty — CHANGES_REQUESTED** [review]
We usually don't recommend new keyboard submission with large userspace updates, but since it's a handwired board it isn't a big deal.  I left a few suggestions regarding convention used on the ortho36.

File: keyboards/handwired/ortho36/rules.mk [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+# OPT_DEFS += -DINTERRUPT_CONTROL_ENDPOINT
+
+
+# Boot Section Size in *bytes*
+#   Teensy halfKay   512
+#   Teensy++ halfKay 1024
+#   Atmel DFU loader 4096
+#   LUFA bootloader  4096
+#   USBaspLoader     2048
+# OPT_DEFS += -DBOOTLOADER_SIZE=512
```
Comment (drashna) [inline keyboards/handwired/ortho36/rules.mk:50]:
```suggestion
```

Though, you will want to define the `BOOTLOADER`

File: keyboards/preonic/keymaps/alysaght/config.h [hunk 4]
```diff
@@ -0,0 +1,34 @@
+#pragma once
```
Comment (drashna) [inline keyboards/preonic/keymaps/alysaght/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: users/alysaght/alysaght.c [hunk 5]
```diff
@@ -0,0 +1,115 @@
+#include "alysaght.h"
```
Comment (drashna) [inline users/alysaght/alysaght.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: users/alysaght/alysaght.h [hunk 6]
```diff
@@ -0,0 +1,350 @@
+#pragma once
```
Comment (drashna) [inline users/alysaght/alysaght.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: users/alysaght/config.h [hunk 7]
```diff
@@ -0,0 +1,35 @@
+#define TAPPING_TERM 215
```
Comment (drashna) [inline users/alysaght/config.h:1]:
```suggestion
#pragma once

#define TAPPING_TERM 215
```

Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by ALysaght** [comment]
@ridingqwerty and @drashna thank you both for the feedback, apologies for my complete lack of understanding of C++. I believe each of your requests have been addressed, let me know if additional changes are needed.

File: keyboards/dztech/dz65rgb/keymaps/alysaght/keymap.c [hunk 8]
```diff
@@ -0,0 +1,21 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/dztech/dz65rgb/keymaps/alysaght/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/handwired/ortho36/ortho36.c [hunk 9]
```diff
@@ -0,0 +1,5 @@
+#include "ortho36.h"
```
Comment (drashna) [inline keyboards/handwired/ortho36/ortho36.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/handwired/ortho36/ortho36.c [hunk 10]
```diff
@@ -0,0 +1,5 @@
...
+
+void matrix_init_kb(void) {
+	matrix_init_user();
+};
```
Comment (drashna) [inline keyboards/handwired/ortho36/ortho36.c:5]:
```suggestion
```

File: keyboards/handwired/ortho36/ortho36.h [hunk 11]
```diff
@@ -0,0 +1,20 @@
+#ifndef PLANCK_H
+#define PLANCK_H
```
Comment (drashna) [inline keyboards/handwired/ortho36/ortho36.h:2]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

```suggestion
#pragma once
```

File: keyboards/handwired/ortho36/ortho36.h [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+) \
+{ \
+	{ k00, k01, k02, k03, k04, k05, k06, k07, k08, k09 }, \
+	{ k10, k11, k12, k13, k14, k15, k16, k17, k18, k19 }, \
+	{ k20, k21, k22, k23, k24, k25, k26, k27, k28, k29 }, \
+	{ k30, k31, k32, k33, k34, k35, k36, k37, k38, k39 } \
+}
+
+
+#endif
```
Comment (drashna) [inline keyboards/handwired/ortho36/ortho36.h:20]:
```suggestion
```

File: keyboards/keebio/nyquist/keymaps/alysaght/keymap.c [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+)
+
+
+};
+
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
```
Comment (drashna) [inline keyboards/keebio/nyquist/keymaps/alysaght/keymap.c:95]:
```suggestion
```

File: users/alysaght/config.h [hunk 14]
```diff
@@ -0,0 +1,37 @@
+#pragma once
```
Comment (drashna) [inline users/alysaght/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**